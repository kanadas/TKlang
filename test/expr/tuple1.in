(\x -> if x == (1, true, 2, false) then 1 else 2) (1, true, 2, false)
