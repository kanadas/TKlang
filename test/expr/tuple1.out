expr/tuple1.in
1
